File: ProjetoB/ProjetoB/Models/Folha.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProjetoB.Models
{
    public class Folha
    {
        public int mes { get; set; }
        public string ano { get; set; }
        public int horas { get; set; }
        public int valor { get; set; }
        public double bruto { get; set; }
        public double irrf { get; set; }
        public double inss { get; set; }
        public double fgts { get; set; }
        public double liquido { get; set; }
        public Funcionario funcionario { get; set; }
    }

    public class Funcionario
    {
        public string nome { get; set; }
        public string cpf { get; set; }
    }
}

File: projeto-a/projeto-a/src/models/folha.ts
export type Folha = {
  mes: number;
  ano: number;
  horas: number;
  valor: number;
  bruto: number;
  irrf: number;
  inss: number;
  fgts: number;
  liquido: number;
  funcionario: Funcionario;
};

type Funcionario = {
  nome: string;
  cpf: string;
};
